k8s/api.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: api
+      app.kubernetes.io/instance: api
+  strategy:
+    type: RollingUpdate
```

Comment:
You can remove this part
```
strategy:
    type: RollingUpdate
```


https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy says that `RollingUpdate` is the default value

Reply:
Didn't know. Thanks.